src/GravityFormsExtensions.php

```diff
@@ -176,6 +199,17 @@ public function p4_gf_form_settings(array $fields): array
             'choices' => self::P4_GF_TYPES,
         ];
 
+        $fields['p4_options']['fields'][] = [
```

Comment:
I'm wondering if there is a way to get all available counter names from the API and present them in a select component? Would avoid potential mistakes and typos

Reply:
I had the same thought and I'm 100% with you.
I checked the API docs and found nothing about that.

Reply:
Maybe @comzeradd knows whether that's possible or not

Reply:
I don't see something like that in the documentation. I'll check with Coding team. But in any case, we can explore that as a follow up, because it may require some additional work on our end too. For instance we may need some auto-completion. There are already 60 counters there and offering a dropdown can be difficult to use.